compiler/dyno/include/chpl/uast/OpCall.h

```diff
@@ -83,6 +83,9 @@ class OpCall final : public Call {
   bool isUnaryOp() const { return children_.size() == 1; }
 };
 
+// Returns true if the given string is an operator name
+bool isUstrOpName(UniqueString name);
```

Comment:
can we just name it `isOpName` ? The "Unique string" part is clear from the type of the formal.